random1on1/random1on1bot.py

```diff
@@ -0,0 +1,45 @@
+import discord
+import logging
+import random1on1
+
+
+def read_config(location):
+    with open(location, "r") as config_file:
+        config = random1on1.api.config.Config(config_file.read())
+    return config
+
+
+class Random1on1Bot(discord.Client):
+
+    logger = logging.getLogger("discord")
```

Comment:
I like `absl` because you don't need this line :)

Reply:
I want to use the same logging instance that the discord.py library uses so that all the logs are integrated in one place

In particular, we can build int a logging destination for this that's a discord channel and send logs to that discord channel so you don't have to dig through the logs on a GitHub actions container when things fail

If we replace stuff with absl, I want to make sure we keep that possibility for sticking logs directly into a discord channel